play-scala-anorm-example/conf/application.conf

```diff
@@ -5,6 +5,9 @@
 db.default.driver=org.h2.Driver
 db.default.url="jdbc:h2:mem:play"
 
+# Evolute 'default' database by default
+applyEvolutions.default=true
```

Comment:
This config does not even exist (see https://github.com/playframework/playframework/blob/main/persistence/play-jdbc-evolutions/src/main/resources/reference.conf)
It doesn't do anything...

Reply:
The config does exist and do something but deprecated. (See test for that https://github.com/playframework/playframework/blob/18b6720d6a395f6a94b47804a3db5368cada44ef/persistence/play-jdbc-evolutions/src/test/scala/play/api/db/evolutions/DefaultEvolutionsConfigParserSpec.scala#L60-L62)

And it's replaced by the option `autoApply` in the file you refer (https://github.com/playframework/playframework/blob/main/persistence/play-jdbc-evolutions/src/main/resources/reference.conf)

I just update the config in the second commit.